src/wally.rs

```diff
@@ -230,9 +230,10 @@ pub fn tx_get_elements_signature_hash(
     sighash: u32,
     segwit: bool,
 ) -> sha256d::Hash {
-    let flags = match segwit {
```

Comment:
Heh, yeah I do that because it's a line shorter and the logic is trivial. Anyhow, this is ok.